lib/hdcontroller.js

```diff
@@ -190,7 +189,7 @@ class HDProxydClient {
                     ._failover(method, stream, size, key, counter, log, callback, params);
             }
             receivedResponse = true;
-            log.end().debug('request received response');
+            log.debug('request received response');
```

Comment:
why remove the `end()` call?

Reply:
because there's no log.end().debug() , we can either have a log.end(msg) or log.debug(msg) , for consistency I went with the log.debug

Reply:
i don't think this is the sementically equivalent : end() has some special meaning, marking the end of a request, so it needs to be called...

Reply:
Indeed done here : https://github.com/scality/hdclient/pull/833/files#diff-775216d7c27f1c276497365c717d78306c34cf8f2b76ef4f0483b3bc86ee203eR238

Reply:
yet we probably don't want to display this in "default" log lvel (info i guess?)
So the equivalent code would be this:

```suggestion
log.debug('request received response');
log.end();
```

That said, looking at werelog code this seems to be working. In class `RequestLogger`, the `end()` called with no parameters method [returns an object of type EndLogger](https://github.com/scality/werelogs/blob/development/8.2/lib/RequestLogger.js#L361) :
```
    end(msg, data) {
        if (msg === undefined && data === undefined) {
            return this.endLogger;
        }
        return this.log(this.endLevel, msg, data, true);
    }
```

and this endLogger does indeed have a [debug method](https://github.com/scality/werelogs/blob/development/8.2/lib/RequestLogger.js#L70).

Reply:
@benzekrimaha ?

Reply:
the fix was done here : https://github.com/scality/hdclient/pull/833/commits/99a0a48dd9c3c5d477d91793d1449c71a63a14b1 , please note that the werelogs version will be bumped once released with the fix